### ISSUE TITLE
Webpack Example

### ISSUE THREAD

**Issue by rodrigojcmello** [johanholmerin/style9#19]
I know that this is not the ideal place for this... I'm having trouble running a webpack with babel-loader. Would you have a more complete example of a configured webpack? I was not successful, following the documentation.

My webpack:

```js
module.exports = (env = { NODE_ENV: 'development' }) => {
  const prod = env.NODE_ENV === 'production';

  // eslint-disable-next-line no-console
  console.log(`MODE: ${prod ? 'PRODUCTION' : 'DEVELOPMENT'}`);

  return {
    entry: './src/index.tsx',
    output: {
      filename: '[name].[chunkhash].js',
      path: path.resolve(__dirname, 'dist'),
    },
    mode: prod ? 'production' : 'development',
    devtool: 'source-map',
    ...(prod
      ? {}
      : { devServer: { historyApiFallback: true, hot: true, hotOnly: true } }),
    plugins: [
      // HTML Template
      new HtmlWebpackPlugin({ template: './index.html' }),
      new Style9Plugin(),
      new MiniCssExtractPlugin(),
    ].filter(Boolean),
    resolve: {
      extensions: ['.tsx', '.ts', '.js'],
    },
    module: {
      rules: [
        {
          test: /\.([tj])sx?$/,
          use: [
            Style9Plugin.loader,
            {
              loader: 'babel-loader',
              options: {
                presets: [
                  ['@babel/preset-env', { targets: 'last 1 chrome version' }],
                  [
                    '@babel/preset-react',
                    {
                      // Removes the need to import React into JSX files
                      runtime: 'automatic',
                    },
                  ],
                  '@babel/preset-typescript',
                ],
                plugins: [
                  // Fast Refresh
                  !prod && require.resolve('react-refresh/babel'),
                ].filter(Boolean),
                cacheDirectory: true,
              },
            },
          ],
        },
        {
          test: /\.css$/i,
          use: [MiniCssExtractPlugin.loader, 'css-loader'],
        },
        {
          test: /\.(png|jpe?g|gif|woff2?|eot|otf|webp)$/i,
          use: 'file-loader',
        },
      ],
    },
  };
};
```

Error:

<img width="1642" alt="Captura de Tela 2020-12-28 às 22 30 14" src="https://user-images.githubusercontent.com/5174526/103252635-7381a200-4932-11eb-9373-6b2c6797c630.png">

**Comment by johanholmerin** [comment]
It seems that the issue is `flatMap` not existing. Which version of node are you using?

**Comment by rodrigojcmello** [comment]
node 15.5.0 / npm 7.3.0
yarn 2.4.0

**Comment by johanholmerin** [comment]
Could you create a repo with the issue?

**Comment by rodrigojcmello** [comment]
sure

**Comment by rodrigojcmello** [comment]
just for the record, i just tested running with npm 7 instead of yarn, and the error changed. Now I will generate a repo.

<img width="1845" alt="Captura de Tela 2020-12-29 às 08 52 08" src="https://user-images.githubusercontent.com/5174526/103282059-5fb55a80-4989-11eb-9b28-a705bba81c2c.png">

**Comment by rodrigojcmello** [comment]
repo: https://github.com/rodrigojcmello/style9-webpack

I disabled the style9 settings in the webpack just to make the application run. I also commented on the use in "src / components / Button / index.tsx"

**Comment by johanholmerin** [comment]
Was caused by incompatibility with Webpack 5. It should be fixed in 0.5.9.
